config/initializers/setup_mail.rb

```diff
@@ -1,10 +1,11 @@
-ActionMailer::Base.delivery_method = :smtp
-ActionMailer::Base.smtp_settings = {
-  address:        'smtp.sendgrid.net',
-  port:           '587',
-  authentication: :plain,
-  user_name:      ENV['SENDGRID_USERNAME'],
-  password:       ENV['SENDGRID_PASSWORD'],
-  domain:         'heroku.com',
-  enable_starttls_auto: true
+ActionMailer::Base.delivery_method = :smtp	
+ActionMailer::Base.perform_deliveries = true	
```

Comment:
@nnat425 maybe we shouldn't perform deliveries in local and development, because we try a lot of test accounts like test@example.com and we wouldn't want to send emails there.